native/src/hooks/useAnnounceSearchResultsIOS.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { AccessibilityInfo, Platform } from 'react-native'
+
+import { CityModel, ExtendedPageModel } from 'shared/api'
+
+const useAnnounceSearchResultsIOS = (searchResults: ExtendedPageModel[] | CityModel[] | null): void => {
```

Comment:
:wrench: Could you perhaps change this to generic arrays?
```suggestion
const useAnnounceSearchResultsIOS = (searchResults: unknown[] | null): void => {
```